homeassistant/components/generic_thermostat/strings.json

```
@@ -17,8 +17,8 @@
         "data_description": {
           "ac_mode": "Set the actuator specified to be treated as a cooling device instead of a heating device.",
           "heater": "Switch entity used to cool or heat depending on A/C mode.",
-          "target_sensor": "Temperature sensor that reflect the current temperature.",
-          "min_cycle_duration": "Set a minimum amount of time that the switch specified must be in its current state prior to being switched either off or on. This option will be ignored if the keep alive option is set.",
+          "target_sensor": "Temperature sensor that reflects the current temperature.",
+          "min_cycle_duration": "Set a minimum amount of time that the switch specified must be in its current state prior to being switched either off or on. This option will be ignored if the `keep_alive` option is set.",
```

Hmm. This was copied from yaml docs before we pruned some settings. We can't even set keep alive from config flow can we? If se we should remove this mention.

---

Yup, we should remove this.

```suggestion
          "min_cycle_duration": "Set a minimum amount of time that the switch specified must be in its current state prior to being switched either off or on.",
```